### PR TITLE
Post-release-changes

### LINKED ISSUE #1978: Stop publishing documentation to old nimi-python readthedocs project.
### Description of issue

We have new, individual Read the Docs projects and the index for latest of the old one has links to them. We don't need to continue building the old project. We should update that project to not build new versions and delete the configuration files, .rsts specific to it.

### PR DESCRIPTION
**Pull request by ni-jfitzger**
- [X] This contribution adheres to [CONTRIBUTING.md](https://github.com/ni/nimi-python/blob/master/CONTRIBUTING.md).
- [X] I've updated [CHANGELOG.md](https://github.com/ni/nimi-python/blob/master/CHANGELOG.md) if applicable.
- ~[ ] I've added tests applicable for this pull request~

Note: In addition to the usual post-release changes, this PR also updates the documentation for the old Read the Docs project in the following ways:
* Update index.rst to point at stable versions of new documentation.
* Stop pulling in files other than index.rst (which users could find through banner links in previous versions)

### What does this Pull Request accomplish?
Get the repo ready for new development after 1.4.5 release.

### List issues fixed by this Pull Request below, if any.
None

### What testing has been done?
PR Checks

### CODE REVIEW

File: docs/index.rst [hunk 1]
```diff
@@ -1,33 +0,0 @@
...
-* niscope
-* niswitch
-* nise
-* nitclk
-
-Simply select the version you wish to view documentation for.
-
-For newer documentation, refer to the individual **Read the Docs** projects.
-
-* `nidcpower <https://nidcpower.readthedocs.io/en/latest>`_
```
Comment (ni-jfitzger) [inline docs/index.rst:25]: I think I should update this to stable and manually rebuild one last time, before I delete.

Comment (ni-jfitzger) [reply docs/index.rst:25]: Hmm, it may be too late to update the link for the stable version and version 1.4.5 pages.

Comment (ni-jfitzger) [reply docs/index.rst:25]: Reverted and updated the links.
I actually already deleted the webhook for the old project. I'll probably need to:
1. Recreate the webhook
2. Delete the version tag from the 1.4.5 release
3. Create a 1.4.5 tag for this PR Commit
4. Delete the webhook after the old project documentation builds (should rebuild 1.4.5, stable and latest to list links to new stable docs)
5. Delete the 1.4.5 tag
6. Readd the 1.4.5 tag to the 1.4.5 release

**Comment by ni-jfitzger** [comment]
I discovered that old nimi-python Read the Docs documentation has links to hidden pages for latest/master/stable documentation that users can be directed to by the "this is not the latest version" banner.

Example Page: https://nimi-python.readthedocs.io/en/1.4.4/nidcpower/class.html

Through this link, they could be directed to what the project thinks is the latest version and see something other than our list of redirects, reading what appears to be documentation for the latest version (albeit poorly built).

To prevent this, we'll need to update the conf.py to not pull in other files.

**Review by marcoskirsch — APPROVED** [review]

File: docs/index.rst (hunk 1 above)
Comment (ni-jfitzger) [reply docs/index.rst:25]: This is done.

**Pull request merged**